### PR TITLE
update deps[patch]

### PR DESCRIPTION
**Pull request by TomPridham**
## Further Information (screenshots, bug report links, etc)
update everything to latest. im getting prettier lint errors in newsada, so im hoping that updating this will fix that

### CODE REVIEW

**Review by jhelst — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     ]
   },
   "peerDependencies": {
-    "eslint": "^6.0.1"
+    "eslint": "^6.2.2"
```
Comment (zacanger) [inline package.json]: If you're using `ncu`, make sure you filter deps. `ncu --dep dev` for example. peerDeps should always be kept on the lowest semver-compatible version that works.

File: package.json [hunk 2]
```diff
@@ -43,27 +43,27 @@
...
+    "eslint-plugin-jest": "22.15.2",
     "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-node": "9.1.0",
     "eslint-plugin-prettier": "3.1.0",
     "eslint-plugin-promise": "4.2.1",
-    "eslint-plugin-react": "7.14.2",
-    "eslint-plugin-react-hooks": "1.6.1",
-    "eslint-plugin-unicorn": "9.1.1"
+    "eslint-plugin-react": "7.14.3",
+    "eslint-plugin-react-hooks": "2.0.1",
```
Comment (zacanger) [inline package.json]: This was technically a breaking change. https://github.com/facebook/react/pull/16455/files

File: package.json (hunk 1 above)
Comment (zacanger) [reply package.json]: up to you

File: package.json (hunk 2 above)
Comment (zacanger) [reply package.json]: we might not, internally, but this is a public package

**Pull request merged**